docs/ru/guide/plugins-and-presets.md

```diff
@@ -112,15 +105,15 @@ vue add vuex
 ``` json
 {
   "useConfigFiles": true,
-  "router": true,
```

Comment:
Though we now have a standalone router plugin, the legacy preset format (`router` and `routerHistoryMode` option) can and should still be supported, so as to reduce the migration costs.

Reply:
I added support for legacy router option but have it removed from docs completely.